Support for counting idling resource aware rxjava scheduler

athkalia opened the pull request.

Commit:
Support for counting idling resource aware rxjava scheduler

miguelslemos commented on app/build.gradle:
```diff
@@ -220,7 +220,7 @@ dependencies {
     // MVP library
     compile "com.hannesdorfmann.mosby3:mvp:$versions.mosbyMvpVersion"
-    compile "com.hannesdorfmann.mosby3:viewstate:$versions.mosbyMvpVersion"
+    compile "com.hannesdorfmann.mosby3:viewstate:3.0.1" //freaky bug won't work with a non-hardcoded version
```
its works for me 
```
// MVP library
    compile "com.hannesdorfmann.mosby3:mvp:$versions.mosbyMvpVersion"
    compile "com.hannesdorfmann.mosby3:viewstate:$versions.mosbyMvpVersion"
    compile "com.hannesdorfmann.mosby3:mvp-nullobject-presenter:$versions.mosbyMvpVersion"
```

athkalia replied:
I tried with your piece of code and it works now. Thanks! I had the exact same thing, something reallly weird must have been happening

miguelslemos commented on app/config/gradle/versioning.gradle:
```diff
@@ -23,7 +23,7 @@ ext.versions = [
         timberVersion             : '4.5.1',
-        mosbyMvpVersion           : '3.0.0',
+        mosbyMvpVersion           : '3.0.1',
         annotationsVersion        : '2.0.1',
```
a new versions is available 3.0.2

athkalia replied:
👍  updated

Commit:
Support for counting idling resource aware rxjava scheduler.
Updated based on PR comments #57.

Commit:
Support for counting idling resource aware rxjava scheduler.
Fixed a test that was failing.
Also ended up initializing jodatime in app as I had forgotten to do it earlier (it loads some timezones and listens to timezone changes)
